trunk/SUAVE/Components/Energy/Networks/Stopped_Rotor.py

```diff
@@ -129,23 +129,23 @@ def evaluate_thrust(self,state):
         num_lift          = self.number_of_engines_lift
         num_forward       = self.number_of_engines_forward
         
-        ###
-        # Setup batteries and ESC's
-        ###
+        ##
+        # SETUP BATTERIES AND ESC's
+        ##
         
         # Set battery energy
         battery.current_energy = conditions.propulsion.battery_energy    
         
-        volts = state.unknowns.battery_voltage_under_load   
+        volts = state.unknowns.battery_voltage_under_load
         volts[volts>self.voltage] = self.voltage
         
         # ESC Voltage
         esc_lift.inputs.voltagein    = volts      
         esc_forward.inputs.voltagein = volts 
         
-        ###
-        # Evaluate thrust from the forward propulsors
-        ###
+        ##
+        # EVALUATE THRUST FROM FORWARD PROPULSORS 
```

Comment:
i like this convention....all uppercase.  We should be aware to make this change in future pull 
(imo)